modules/installing-aws-load-balancer-operator.adoc

```diff
@@ -20,7 +20,6 @@ You can install the AWS Load Balancer Operator by using the web console.
 
 . Navigate to *Operators* → *OperatorHub* in the {product-title} web console.
 . Select the *AWS Load Balancer Operator*. You can use the *Filter by keyword* text box or use the filter list to search for the AWS Load Balancer Operator from the list of Operators.
-. Select the `aws-load-balancer-operator` namespace.
```

Comment:
Do not remove this step.

Reply:
Agree if [the proposed detail](https://github.com/openshift/openshift-docs/pull/76161/files#r1622542453) will be added.